src/getCoinCombination.test.js

```diff
@@ -7,4 +7,44 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
```

Comment:
should be wrapped 
 expect(getCoinCombination)
      .toBeInstanceOf(Function);
  });